yarn.lock

```diff
@@ -1,6590 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
could you add `yarn.lock` to the `.gitignore` then, please?

Reply:
Fixed in e8b5fcf.